coap/Kconfig

```diff
@@ -102,6 +102,12 @@ menu "CoAP Configuration"
 
             If this option is disabled, redundant CoAP Observe Persist code is removed.
 
+    config COAP_Q_BLOCK
+        bool "Enable block-wise transfer in CoAP"
+        default n
+        help
+            Enable Block-Wise Transfers support for CoAP
```

Comment:
Enable Q-Block (RFC9177) fast block transfers using NON (instead of CON) support for CoAP

Reply:
Change is good